pkg/network/tracer/tracer.go

```diff
@@ -182,6 +182,11 @@ func newTracer(cfg *config.Config, telemetryComponent telemetryComponent.Compone
 	tr.reverseDNS = newReverseDNS(cfg, telemetryComponent)
 	tr.usmMonitor = newUSMMonitor(cfg, tr.ebpfTracer)
 
+	if cfg.EnableProcessEventMonitoring || cfg.EnableUSMEventStream {
```

Comment:
I don't think you need this change as the `events` package only gets used by the network consumer.

Reply:
So I need to create a different marker to say "usm is enabled and running" to register my consumer